src/include/network/postgres_network_commands.h

```diff
@@ -0,0 +1,86 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Terrier
+//
+// postgres_network_commands.h
+//
+// Identification: src/include/network/postgres_network_commands.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+// #pragma once
```

Comment:
I think we do want to be able to parse packets correctly in the initial port to make sure the state machines and everything is working. Basically all the commands should be stubs that return immediately, but everything above should be fully functional.

Reply:
Right now I made sure that the layer can read packets sent via ftp to port 2888 and reply with a generic message. Are you saying I plop this dummy functionality into the network commands?